### PR TITLE
NAS-110140 / 12.0 / fix ProcessPoolExector in main.py for python3.9

### PR DESCRIPTION
**Pull request by yocalebo**

### CODE REVIEW

**Comment by yocalebo** [comment]
Note do not merge until these dependent PRs have been merged.

1. https://github.com/truenas/ports/pull/995
2. https://github.com/truenas/core-build/pull/278

**Review by sonicaj — APPROVED** [review]

**Pull request merged**
